graphQL/index.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+    <p>
+        <b>API Национального банка для получения официального курса белорусского рубля по отношению
+            к иностранным валютам.</b>
+    </p>
+    <select id="mode">
+        <option value="1">Курс валюты за период</option>
+    </select>
+    <span id="Cur">Cur_ID валюты 1 <input type="text" id="iCur" style='width:50px' /> Cur_ID валюты 2 <input type="text"
+            id="iCur2" style='width:50px' /> </span>
+    <span id="period">с: <input type="text" id="ifrom" /> по: <input type="text" id="ito" /></span>
+    <input type="button" id="btn" value="получить" />
+    <table id="res" style='width:300px;'>
+        <tr id="head">
+            <td id='date' style="width:80px;"></td>
+            <td id='USD' style="width:50px"></td>
+            <td id='EUR' style="width:50px"></td>
+        </tr>
+    </table>
+    <div id='nodata'></div>
+
+    <script type="text/javascript">
+        const uri = 'https://www.nbrb.by/api/'
+
+        function parseRuDate(s) {
+            let parts = s.split('.');
+
+            parts[0] = parseInt(parts[0], 10);
+            parts[1] = parseInt(parts[1], 10);
+            parts[2] = parseInt(parts[2], 10);
+
+            return `${parts[2]}-${parts[1]}-${parts[0]}`
+        }
+
+        function currency() {
+            fetch(uri + 'exrates/currencies/' + document.querySelector('#iCur').value)
+                .then(response => {
+                    if (response.status >= 200 && response.status < 400) {
+                        return response.json()
+                    }
+                })
+                .then(data => {
+                    typeof data === "string" ? JSON.parse(data) : data
+                    if (data === undefined) {
+                        document.querySelector('#USD').innerHTML = 'NOT FOUND'
+                    } else {
+                        document.querySelector('#USD').innerHTML = data.Cur_Abbreviation
+                    }
+
+                    document.querySelector('#date').innerHTML = 'Дата'
+                })
+                .catch(err => {
+                    console.log(err)
+                })
+        }
+
+        function currency2() {
```

Comment:
duplicate, better to merge this two functions and pass currency number by argument